client/components/domains/register-domain-step/index.jsx

```diff
@@ -220,7 +223,7 @@ var RegisterDomainStep = React.createClass( {
 		async.parallel(
 			[
 				callback => {
-					if ( domain.indexOf( '.' ) < 0 ) {
+					if ( ! domain.match( /.{3,}\..{2,}/ ) ) {
```

Comment:
yep, I actually started with it and then changed it to `.{}`. This code only cares about valid domains that we still type, it doesn't care about invalid domains - the endpoint will verify that. So if you type `somedomain`. or `somedomain.c`, we don't render a notice.